libcodechecker/server/client_db_access_handler.py

```diff
@@ -697,27 +697,35 @@ def getSuppressedBugs(self, run_id):
                                               msg)
 
     @timeit
-    def getComments(self, bug_hash):
+    def getComments(self, report_id):
         """
             Return the list of comments for the given bug.
         """
         session = self.__session
         try:
-            result = []
+            report = session.query(Report).get(report_id)
+            if report:
+                result = []
 
-            comments = session.query(Comment) \
-                .filter(Comment.bug_hash == bug_hash) \
-                .order_by(Comment.created_at.desc()) \
-                .all()
+                comments = session.query(Comment) \
+                    .filter(Comment.bug_hash == report.bug_id) \
```

Comment:
Hash and ID is confusing in terminology, I've put a separate issue about this #763. Just mentioning it here, no action to take regarding this pull request.